package.json

```diff
@@ -26,7 +26,8 @@
     "@actions/http-client": "^2.0.1",
     "@actions/io": "^1.1.2",
     "@actions/tool-cache": "^2.0.1",
-    "@octokit/webhooks-definitions": "^3.67.3"
+    "@octokit/webhooks-definitions": "^3.67.3",
+    "tsc": "^2.0.4"
```

Comment:
[`tsc`](https://www.npmjs.com/package/tsc) is not an official Microsoft package, should be removed from the list of dependencies here. The `tsc` binary is already provided by the `typescript` dev dependencies.

Reply:
The `tsc` addition can be reverted with:

```sh
yarn remove tsc
yarn run all
rm .licenses/npm/tsc.dep.yml
```